src/OrchardCore.Modules/OrchardCore.Search.Azure.CognitiveSearch/AdminMenu.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Localization;
+using OrchardCore.Navigation;
+
+namespace OrchardCore.Search.Azure.CognitiveSearch;
```

Comment:
Why it's not `OrchardCore.Search.Azure` or `OrchardCore.Search.Azure.CognitiveSearch`?

Reply:
Azure could have many search services

Reply:
Sure, while the context is search there's no confusion, moreover, after the rebranding we might use `OrchardCore.Search.AzureAI`

Reply:
Not a fan of `OrchardCore.Search.AzureAI`, but can't think of a better name. So I used it.